widget/src/components/messages/FileMessage.tsx

```
@@ -23,19 +22,34 @@ const FileMessage: React.FC<FileMessageProps> = ({ message }) => {
-
+  if (message.data.type === "unknown") {
+    return (
+      <div
+        className="sc-message--file"
+        style={{
+          color: colors.text,
+          backgroundColor: colors.bg,
+        }}
+      >
+        <p className="error-message">{t("messages.unsupported_file_type")}</p>;
+      </div>
```

Biome (1.9.4)

[error] There is a suspicious semicolon in the JSX element.

This is usually the result of a typo or some refactor gone wrong.
Remove the semicolon, or move it inside a JSX element.

(lint/suspicious/noSuspiciousSemicolonInJsx)